lib/medtronic/processData.js

```diff
@@ -426,6 +464,84 @@ function buildBGRecords(records) {
   return postrecords;
 }
 
+function buildCGMRecords(events) {
+
+  var postrecords = [];
+  var start = null;
+  var recordsSinceTimestamp = null;
+
+  // we have to work through the CGM records in reverse, as sensor timestamps
+  // affect previous values and are subject to time changes
+
+  for( var i = events.length - 1 ; i >= 0; i-- ) {
+    var event = events[i];
+
+    if(event.jsDate != false) {
+      console.log('CGM event:',sundial.formatDeviceTime(event.jsDate), event.type.name, common.bytes2hex([event.head]),common.bytes2hex(event.date),common.bytes2hex(event.body),common.bytes2hex(event.descriptor) );
```

Comment:
Yes, eventually. At the moment there are lots of `console.log`s just to make things easier to debug during alpha testing. I plan on removing most of them before going to prod, or change them to only show up if the app is built in debug mode.